lib/modules/manager/nuget/util.ts

```diff
@@ -141,6 +141,18 @@ export async function getConfiguredRegistries(
     }
   }
 
+  // Deduplicate registries with #procolVersion=3
+  // Keep any which include sourceMappedPackagePatterns
+  const plainRegistryUrls = registries
+    .filter((r) => !r.sourceMappedPackagePatterns)
+    .map((r) => `${r.url}`);
```

Comment:
```suggestion
    .map((r) => r.url);
```

it's already a string (perf)